package.json

```diff
@@ -16,7 +16,7 @@
     "build": "hugo --minify",
     "test": "npm run lint:markdown && npm run lint:css && npm run lint:js && npm run test:links",
     "pretest:links": "npm run build",
-    "test:links": "linkinator ./public --recurse --skip '^https?://(www.linkedin.com|twitter.com|localhost:3000/api/)' --verbosity error",
+    "test:links": "linkinator ./public --recurse --skip '^https?://(www.linkedin.com|twitter.com|localhost:3000/api/|stackoverflow.com|github.com/organizations/OpenTermsArchive/settings/personal-access-token-requests)' --verbosity error",
```

Comment:
I note here that we should use the same principle as https://github.com/OpenTermsArchive/opentermsarchive.org/commit/f334691b82043c3b591078953bb5a791dfecc72f to avoid having to manage this list of exceptions.

Reply:
What do you mean?

Reply:
As part of the work in progress to migrate the princinpal site to Hugo, we've changed the way link testing works so that we no longer have to manage an exception list. As we're being dumped by Cloudflare, we've stopped testing external links in CI. We should do the same thing on the doc site ;)